mnefun/_sss.py

```diff
@@ -411,6 +412,19 @@ def run_sss_locally(p, subjects, run_indices):
         if p.disp_files:
             print('    Maxwell filtering subject %g/%g (%s).'
                   % (si + 1, len(subjects), subj))
+        # compute eSSS basis
+        kwargs = dict()
+        if p.cont_as_esss:
+            assert p.proj_meg == 'combined', 'must be combined for eSSS'
+            proj_nums = _proj_nums(p, subj)[2]
+            if any(proj_nums):
+                prebad_file = (p, subj, dict(autobad=False, prefix=' ' * 8))
+                print('      Computing eSSS basis from ERM with %d components'
+                      % (proj_nums[0],))
+                kwargs['extended_proj'] = _compute_erm_proj(
+                    p, subj, projs=[], kind='raw', bad_file=prebad_file,
+                    remove_existing=True, disp_files=False)
+
```

Comment:
It would be nice here (maybe within _computer_erm_proj()) to have a printed confirmation of the projector file name saved (_prepoc_cont....fif_).

Reply:
Also, I was wrong about there being an error with bad channels (such that prebads were loaded from a .txt file but not via the yml parameters).  I believe I originally misinterpreted a section in __ssp.py_ involving **postbad** channels, which as you said are usually just for EEG channels.

Reply:
> o have a printed confirmation of the projector file name saved (prepoc_cont....fif).

For this it's a standard path and most people most of the time shouldn't care, so I'd rather not add this to the printed outputs.